### PR TITLE
Clean up resources, if CRD was removed before completion of spark-submit

### PR DESCRIPTION
**Pull request by AlexeyRumyantsev**
Fix issue, when driver is created after CRD is removed.

Sequence:
1. CRD is created
2. operator started spark-submit
3. CRD is removed
4. operator perform cleanup logic
5. spark-submit managed to create driver pod at this moment
6. Driver pod is in an unmanaged state

This pull request adding double check if CRD is still exist after submitting application

### CODE REVIEW

File: pkg/controller/sparkapplication/controller.go [hunk 1]
```diff
@@ -551,6 +551,21 @@ func (c *Controller) syncSparkApplication(key string) error {
 		} else {
 			appCopy = c.submitSparkApplication(appCopy)
 		}
+		// Remove just submitted application, if CRD doesn't exist anymore
+		appVerificationCopy, err := c.getSparkApplication(namespace, name)
+		if err != nil {
```
Comment (staceystaceybangbang007) [inline pkg/controller/sparkapplication/controller.go:556]: Dnfjxmf .  I.  Jfncd k f co n co n dkcc.   Jfngkxr.  Cold no I I j. X I.  J j did be x. Dkkx d mmm mmm in fbk cj. C j. Kk co j jcn my CV for hi I'm.  Nik cc.  J

**Review by staceystaceybangbang007 — APPROVED** [review]

**Comment by AlexeyRumyantsev** [comment]
Hi @liyinan926 , sorry for bothering, can you help to check if this contribution can be merged?

**Comment by AlexeyRumyantsev** [comment]
Hi @liyinan926 ! Can you help to review?

**Comment by AlexeyRumyantsev** [comment]
@liyinan926 Liyinan, please help to review

**Pull request closed without merging**